### PR TITLE
Don't merge yet

### PR DESCRIPTION
**Pull request by jiyunyao**
Thanks for opening a Pull Request!

Here's a handy checklist to ensure your PR goes smoothly.

- [ ] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)
- [ ] My code conforms to Google's [python style](https://google.github.io/styleguide/pyguide.html).
- [ ] My PR at a minimum doesn't decrease unit-test coverage (if applicable).
- [ ] All of the [unit-tests](http://forsetisecurity.org/docs/development/#executing-tests) still pass
- [ ] Running `pylint --rcfile=pylintrc` passes.

These guidelines and more can be found in our [contributing guidelines](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: google/cloud/security/iam/explain/utils.py [hunk 1]
```diff
@@ -7,9 +7,40 @@
 #    http://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
+# ditime_stributed under the License is ditime_stributed on an "AS IS" BASIS,
```
Comment (blueandgold) [inline google/cloud/security/iam/explain/utils.py]: fix this?

File: google/cloud/security/iam/explain/utils.py [hunk 2]
```diff
@@ -7,9 +7,40 @@
...
+        elif time_str[23] == '-':
+            parsed_time += datetime.timedelta(hours=int(time_str[24:26]),
+                                              minutes=int(time_str[27:29]))
+    else:
+        raise Exception('The time format is different: '+time_str)
+    return parsed_time
+
+def _json_escape(json_str):
+    """ To deal with potential \n escape in the json string"""
+    return json_str.replace('\\\n', '\\n').replace('\n', '\\n')
```
Comment (blueandgold) [inline google/cloud/security/iam/explain/utils.py]: This really works?  Seems you are going back to the original ```\\\n``` ???

File: google/cloud/security/iam/explain/utils.py (hunk 1 above)
Comment (jiyunyao) [inline google/cloud/security/iam/explain/utils.py]: Sure, that is a stupid replace... I will avoid using replace

File: google/cloud/security/iam/explain/utils.py (hunk 2 above)
Comment (jiyunyao) [inline google/cloud/security/iam/explain/utils.py]: nope, it won't, "\\" is a character. "\\n" is "\\"+"n" and contains no "\n"

**Comment by blueandgold** [comment]
Adding `cla: yes` label, as the PR & commit authors are googlers at the time of this PR was created.

**Pull request closed without merging**